### PR TITLE
[CoW] Migrate trade_slippage

### PR DESCRIPTION
**Pull request by bh2smith**
Relatively easy table migration. No query changes necessary.

PoC - https://github.com/duneanalytics/spellbook/pull/3749

cc @gentrexha

### CODE REVIEW

**Review by gentrexha — APPROVED** [review]

File: models/cow_protocol/ethereum/cow_protocol_ethereum_trade_slippage_legacy.sql [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {{ config(
-	tags=['legacy'],
```
Comment (Hosuke) [inline models/cow_protocol/ethereum/cow_protocol_ethereum_trade_slippage_legacy.sql:2]: Not sure if I need to remove this tag.

**Review by Hosuke — APPROVED** [review]
Seems lgtm.✅
Thank you @bh2smith

File: models/cow_protocol/ethereum/cow_protocol_ethereum_trade_slippage_legacy.sql (hunk 1 above)
Comment (bh2smith) [reply models/cow_protocol/ethereum/cow_protocol_ethereum_trade_slippage_legacy.sql:2]: I could put it back..

**Review by jeff-dude — APPROVED** [review]
lgtm!

**Pull request merged**